homeassistant/components/imazu_wall_pad/wall_pad.py

```diff
@@ -0,0 +1,107 @@
+"""Wall Pad device class."""
+from datetime import timedelta
+import logging
+from typing import Generic, TypeVar
+
+from wp_imazu.packet import ImazuPacket
+
+from homeassistant.const import Platform
+from homeassistant.core import callback
+from homeassistant.helpers.dispatcher import async_dispatcher_connect
+from homeassistant.helpers.entity import DeviceInfo
+from homeassistant.helpers.restore_state import RestoredExtraData, RestoreEntity
+from homeassistant.util import Throttle
+
+from . import ImazuGateway
+from .const import (
+    ATTR_DEVICE,
+    ATTR_ROOM_ID,
+    ATTR_SUB_ID,
+    BRAND_NAME,
+    DOMAIN,
+    MANUFACTURER,
+    MODEL,
+    PACKET,
+    SW_VERSION,
+)
+from .helper import host_to_last
+
+_LOGGER = logging.getLogger(__name__)
+
+T = TypeVar("T", bound=ImazuPacket)
+
+MIN_TIME_BETWEEN_UPDATES = timedelta(seconds=90)
+
+
+class WallPadDevice(Generic[T], RestoreEntity):
```

Comment:
New integrations/entities must support [translation of names and attributes](https://developers.home-assistant.io/blog/2023/03/27/entity_name_translations/).

Please adopt your code accordingly.